src/jdk.compiler/share/classes/com/sun/tools/javac/util/StringNameTable.java

```diff
@@ -62,7 +62,12 @@ public StringNameTable(Names names, int initialCapacity, boolean intern) {
 
     @Override
     public Name fromString(String string) {
-        return this.nameMap.computeIfAbsent(string, s -> new NameImpl(this, intern ? s.intern() : s));
+        Name name = this.nameMap.get(string);
```

Comment:
stylistic - do we need the `this.` qualifiers?

Reply:
No, I just went along with pre-existing style here. If there's a preference to avoid redundant `this.` I can remove it.